### ISSUE TITLE
508-defect-4 [SCREENREADER]: Lists that are not styled as a list and with only one list item should not be written as a list

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#27593]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I don't want to listen to an unnecessary list announcement if what I'm reading is not in fact a list.

## Details

Under "need more help" on the other language page, only one help option is provided "MyVA411 main information line: 800-698-2411." Because it is written as an unordered list `ul`, screen reader users may still hear it announced as a list when it is not designed as one.

## Acceptance Criteria

- [ ] Need more help's content is a `p`, not a `ul` OR
- [ ] More help options are added to justify it being a list

## Steps to Recreate

1. Enter `https://www.va.gov/resources/how-to-get-free-language-assistance-from-va/` in browser
2. Inspect the content below "need more help"
3. Confirm it is a `ul`, not a `p`

## Proposed Solution (if known)

Remove the `ul` `li` and place "MyVA411 main information line: 800-698-2411" within a `p` element.
For example:

```html
<p class="vads-u-margin-top--1">
  <strong>MyVA411 main information line: </strong>
  <a href="tel:1-800-698-2411" rel="noreferrer noopener" id="8e32ea4163003e630dacb9e1a9f31c01" lang="en">
    800-698-2411
  </a>
</p>

## Screenshots or Trace Logs

<img width="1596" alt="Screen Shot 2021-07-20 at 11 08 50 AM" src="https://user-images.githubusercontent.com/14154792/126349428-2e2399a7-7d41-43d9-88df-bdb4a1f2227c.png">

**Comment by davidconlon** [comment]
https://design.va.gov/templates/hub#cross-reference--footnote-component

This is essentially something that should be componentized to address this in a reusable way.
